app/controllers/admin/base_admin_controller.rb

```diff
@@ -58,5 +58,10 @@ def clear_session
       session.delete(:role_codes)
       session.delete(:claims_backlink_path)
     end
+
+    def set_claim_summary_view
+      policy_append = @claim.policy.to_s.underscore if @claim.policy.is_a?(Policies::FurtherEducationPayments)
+      @claim_summary_view = ["claim_summary", policy_append].compact.join("_")
+    end
```

Comment:
can this be a view helper? that way there are zero controller changes

Reply:
👍🏽 that is an excellent suggestion